src/dlangui/core/collections.d

```diff
@@ -248,6 +248,17 @@ struct ObjectList(T) {
         assert(index >= 0 && index < _count, "child index out of range");
         return _list[index];
     }
+    /// get item by index. Returns null if item not found
+    inout(T) tryGet(int index) @safe inout nothrow {

```

Comment:
Is `inout` really necessary here?

Reply:
After a quick check - yes, inout is indeed necessary